Make some changes to home and about pages.

Mo7amedSMahdi opened the pull request:
- Add mobile menu slide-in animation.
- Change the title font to cocogoose on the home and about page.
- Change the hero background image.
- Change the title background image to have a red image effect on the title for the home page.
- Implement the mobile menu animation in javascript.
- Add keyframe animation for the mobile menu in main.css.
- Add a border on hover for the program cards.
-
https://www.loom.com/share/a2f19a813b7b4566b7852051bf3f18b1

Mo7amedSMahdi committed:
staging

Mo7amedSMahdi committed:
Merge remote-tracking branch 'origin/main' into main

Mo7amedSMahdi committed:
Add the Full frontend desktop, Tablet and mobile site.

Mo7amedSMahdi committed:
Create about page and Javascript

- Add about page content.
- Add css styles for the about page
- Add speakers dynamically using Javascript to the index page.
- Add functional mobile menu.
- Add descriptive README for the project.

Mo7amedSMahdi committed:
Update README.md

Mo7amedSMahdi committed:
Change the screenshots images

Mo7amedSMahdi committed:
Update README.md

Mo7amedSMahdi committed:
fix linters

Mo7amedSMahdi committed:
Merge branch 'about-page' of https://github.com/Mo7amedSMahdi/JavaScript-Capstone-Project into about-page

Mo7amedSMahdi committed:
Chages required by code reviewer.

-fix header__nav in tablet and disktop screen sizes.
- Change the padding and font family and font colors.
- Change the background of programs and add borders on hover.
- Change programs button on mobile.
- Show only two speakers on mobile and add button to show more.
- Remove partners from about page on disktop and tablet;
- Change footer background color on desktop and tablet.

Mo7amedSMahdi committed:
change brand image alt from mozilla to careem

Mo7amedSMahdi committed:
fix careem image

Mo7amedSMahdi committed:
add footer to main css styles

Mo7amedSMahdi committed:
Merge branch 'schedule-page' into about-page

Mo7amedSMahdi committed:
fix program border hover

Mo7amedSMahdi committed:
Changes on Home and about page.

- Add mobile menu slidein animation.
- Change the title font to cocogoose on home and about page.
- Change the hero background image.
- Change the title background image to have red image effect on title for the home page.
- Implement the mobile menu animaition in javascript.
- Add keyframe animation for the mobile menu in main.css.
- Add border on hover for the program cards.

CollinsTatang commented on index.html:
```diff
@@ -5,46 +5,56 @@
-                <li><a href=""></a></li>
-                <li><a href=""></a></li>
-                <li><a href=""></a></li>
+        <div id="mobileNav" class="hidden">
+            <button id="btnClose" type="button" class="btn btn-icon close hidden" data-target="mobileMenuClose"><i
+                    class="icon-close"></i></button>
+            <ul id="mobileNavMenu" class="nav--mobile">
+                <li class="active"><a href="index.html">Home</a></li>
+                <li><a href="about.html">About</a></li>
+            </ul>
+        </div>
```
- [ ] Please add this bottom to maintain the main [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
![Addbutton](https://user-images.githubusercontent.com/55596970/158559237-5a981e0a-8a81-4eee-8353-5c41ac39b4aa.png)

ShoiraTa replied:
Adressed 💪🏻

CollinsTatang commented on about.html:
```diff
@@ -5,10 +5,123 @@
+        </section>
+    </main>
+
+    <footer class="footer flex">
+        <div class="footer__group">
+            <img src="images/brands/it_live.png" alt="creative commones korea logo">
+        </div>
+        <div class="footer__group span-2">
+            <p>"Like Never Before"</p>
+            <p>2019 The largest Iraqi forum for information technology in all its disciplines.</p>
+        </div>
+    </footer>
```
- [x] Please the about footer page should be center as the home page footer
![cap-footers](https://user-images.githubusercontent.com/55596970/158560319-3de4df4c-3e1a-4643-9e7d-dfab766fcdfc.png)

ShoiraTa replied:
Adressed 👍🏻

CollinsTatang commented on about.html:
```diff
@@ -5,10 +5,123 @@
+            </div>
+        </div>
+        <div id="mobileNav" class="hidden">
+            <button id="btnClose" type="button" class="btn btn-icon close hidden" data-target="mobileMenuClose"><i
```
- [x] Kindly fix the closing button it doesn't work

https://user-images.githubusercontent.com/55596970/158561315-4f83d24f-a0c6-4a9a-8efb-41d83e5249a6.mp4

ShoiraTa replied:
Adressed 👍🏻

CollinsTatang reviewed (changes requested):
Hi @Mo7amedSMahdi , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

Mo7amedSMahdi committed:
Changes required by codereviewer.

- Fix footer on about page.
- Fix careem brand image on about page.
- fix close button.

Mo7amedSMahdi committed:
Fix mobile nav animation and close button.

Mo7amedSMahdi committed:
fix linters

ShoiraTa reviewed (approved):
## **STATUS: APPROVED  🟢 ** 
------
### Hi @Mo7amedSMahdi , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

![5SW](https://user-images.githubusercontent.com/77038610/154919611-7d0675af-6364-45a3-92d6-eccd9b743ced.gif)

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
